packages/web-comment/src/views/includes/body-start.njk

```diff
@@ -0,0 +1,12 @@
+{% block bodyStart %}
+
+  {% if featureFlag.googleTagManager %}
```

Comment:
I don't think this feature flag is defined anywhere?

Reply:
Just resolving for now to get the initial merge done then we can pick these points back up